src/Piston/AtomActionResult.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace Piston
```

Comment:
Not sure if this class should go in a different namespace actually rather than the project root?